libtenzir/builtins/components/metrics_collector.cpp

```diff
@@ -57,6 +58,9 @@ struct metrics_collector_state {
   std::vector<instance> instances = {};
 
   auto setup() -> caf::expected<void> {
+    if (caf::get_or(self->system().config(), "tenzir.enable-metrics", true)) {
+      return {};
+    }
```

Comment:
This disables instead of enabling the metric if the option is set.

In any case, as we discussed this morning, please undo this change, and remove the option in its entirety.